### PR TITLE
Document COS integration

### PR DESCRIPTION
**Pull request by keirthana**
- Add howto guide for COS integration
- Update landing page
- Update index page to include navigation link

### CODE REVIEW

File: howto/monitor/cos.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+Anbox Cloud uses [Canonical Observability Stack(COS)](https://charmhub.io/topics/canonical-observability-stack) to enable users to collect and monitor their workloads.
+
+Since Anbox Cloud uses Juju for deployment, by supporting [COS Lite](https://charmhub.io/topics/canonical-observability-stack/editions/lite) integration, we can export observability information with Prometheus, Loki and Grafana.
+
+Currently, only the [AMS charm](https://charmhub.io/ams) supports COS Lite integration. To gather observability metrics for the AMS charm,
+
+1. Deploy the COS Lite bundle. See the [COS Lite tutorial](https://charmhub.io/topics/canonical-observability-stack/tutorials/install-microk8s) for instructions.
+2. Deploy the AMS charm as described in the [Instrumenting machine charms](https://charmhub.io/topics/canonical-observability-stack/tutorials/instrumenting-machine-charms) tutorial.
```
Comment (morphis) [inline howto/monitor/cos.md:8]: That tutorial is quite confusing as it mixes many things (extending a charm with observability, deploying it and integrating with the cos stack). I am not sure if that serves the need we have here well enough to show how the charm can be related to the COS stack. Wdyt?

File: howto/monitor/cos.md [hunk 2]
```diff
@@ -0,0 +1,8 @@
+Anbox Cloud uses [Canonical Observability Stack(COS)](https://charmhub.io/topics/canonical-observability-stack) to enable users to collect and monitor their workloads.
+
+Since Anbox Cloud uses Juju for deployment, by supporting [COS Lite](https://charmhub.io/topics/canonical-observability-stack/editions/lite) integration, we can export observability information with Prometheus, Loki and Grafana.
+
+Currently, only the [AMS charm](https://charmhub.io/ams) supports COS Lite integration. To gather observability metrics for the AMS charm,
```
Comment (morphis) [inline howto/monitor/cos.md:5]: We have to be careful here as this is only true for an unreleased version of the charm which we do not officially support yet. We need to call this out here.

File: howto/monitor/cos.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+Anbox Cloud uses [Canonical Observability Stack(COS)](https://charmhub.io/topics/canonical-observability-stack) to enable users to collect and monitor their workloads.
```
Comment (morphis) [inline howto/monitor/cos.md:1]: I think we need to differentiate two parts here:

1. General machine monitoring with grafana-agent
2. Individual workload monitoring

As we can not provide 2) for an official release of Anbox Cloud yet, maybe we focus on 1) only?

**Review by morphis — CHANGES_REQUESTED** [review]

File: howto/monitor/cos.md (hunk 1 above)
Comment (jat-canonical) [reply howto/monitor/cos.md:8]: I still think majority of that tutorial is applicable as the customer needs to know about the grafana agent charm to deploy it first and then relate to the anbox charms. The only thing that is not of their use it point 2 which actually mentions changes to charm code which they are not expected to do. But apart from that everything still stands. So we can copy the information from there make it specfic to anbox and remove the information asking them to make changes to charm ?

**Comment by keirthana** [comment]
Closing without merging as per discussion in https://warthogs.atlassian.net/browse/AC-2103. We will reopen this when we can officially support the integration in latest/stable

**Pull request closed without merging**